interpreter/cling/lib/Interpreter/IncrementalExecutor.cpp

```diff
@@ -91,6 +91,19 @@ std::unique_ptr<TargetMachine>
   CodeModel::Model CMModel = CodeModel::JITDefault;
   CodeGenOpt::Level OptLevel = CodeGenOpt::Less;
 
+  const char* OptLevelEnvConfig = std::getenv("ROOT_OPTIMIZATION_LEVEL");
```

Comment:
Can we react on -O0 etc passed to the cling constructor, and pass the -On flag in when constructing the interpreter in core/meta/src/TCling.cxx?